daemon/cmd/daemon_main.go

```diff
@@ -1053,6 +1053,9 @@ func initializeFlags() {
 	flags.Bool(option.EnableBGPControlPlane, false, "Enable the BGP control plane.")
 	option.BindEnv(Vp, option.EnableBGPControlPlane)
 
+	flags.Bool(option.EnablePMTUDiscovery, false, "Enable path MTU discovery to send ICMP fragmentation-needed replies to the client (default: false)")
```

Comment:
We have other options that default to `false` and we don't document them, viper library takes care of documenting which flags are enabled by default without passing them as an argument to the agent.

```suggestion
	flags.Bool(option.EnablePMTUDiscovery, false, "Enable path MTU discovery to send ICMP fragmentation-needed replies to the client")
```

Reply:
Done